farmhelden/rest_server/models.py

```diff
@@ -14,6 +14,14 @@ class User(models.Model):
         ('2', 'Helfer'))
 
     id = models.AutoField(primary_key=True)
+    session_token=models.CharField(max_length=256)
+    email = models.CharField(max_length=256)
+    passkey = models.CharField(max_length = 256) #should be a hash; function TODO
```

Comment:
should be a hash :)

Reply:
Usage-Comment; the field is a char-field ;)